DbService/inc/GrlHeader.hh

```diff
@@ -0,0 +1,39 @@
+#ifndef DbService_GrlHeader_hh
+#define DbService_GrlHeader_hh
+
+// holds a defintion of a goodrun list - name, etc
+// locked==0 is unlocked, locked >0 is locked
+
+#include <iostream>
+#include <string>
+#include <vector>
+
+namespace mu2e {
+
+class GrlHeader {
+ public:
+  typedef std::vector<GrlHeader> HeaderVec;
+  GrlHeader(const std::string& name, int locked = 0, int lid = -1,
```

Comment:
what use case do you imagine requires everything except name have default (non-functional) values? Would it be better to have a default (no argument) constructor, and one that requires all fields be set?

Reply:
you are right, there are no general use cases so I reduced it to more logical pair